src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -134,6 +139,45 @@ public static IEnumerable<LowLevelClientMethod> BuildMethods(RestClientBuilder b
             }
         }
 
+        private LowLevelConvenienceMethod BuildConvenienceMethod(in LowLevelClientMethod clientMethod, RestClientBuilder builder)
+        {
+            List<Parameter> parameters = new List<Parameter>();
+            MethodSignature protocolSignature = clientMethod.Signature;
+            InputOperation operation = clientMethod.RequestMethod.Operation;
+            Parameter? bodyParameter = null;
+            foreach (var parameter in protocolSignature.Parameters)
+            {
+                if (parameter == KnownParameters.RequestContent || parameter == KnownParameters.RequestContentNullable)
+                {
+                    bodyParameter = builder.GetBodyParameter(operation, DefaultNamespace);
+                    parameters.Add(bodyParameter!);
+                }
+                else if (parameter == KnownParameters.RequestContext)
+                {
+                    parameters.Add(KnownParameters.CancellationTokenParameter);
+                }
+                else
+                {
+                    parameters.Add(parameter);
+                }
+            }
+
+            CSharpType? responseType = builder.GetResponseType(operation, DefaultNamespace);
```

Comment:
If you don't use `GetResponseType` anywhere else, I'd move it into `LowLevelClient`, cause it isn't used for REST operation logic

Reply:
No, but the problem is `LowLevelClient` does have a `TypeFactory`.